tests/e2e/upgrade/wdio.conf.js

```diff
@@ -23,7 +23,7 @@ const UPGRADE_SERVICE_DOCKER_COMPOSE_FOLDER = utils.makeTempDir('upgrade-service
 const CHT_DOCKER_COMPOSE_FOLDER = utils.makeTempDir('cht-');
 const CHT_DATA_FOLDER = utils.makeTempDir('cht-');
 const UPGRADE_SERVICE_DC = path.join(UPGRADE_SERVICE_DOCKER_COMPOSE_FOLDER, 'cht-upgrade-service.yml');
-const MAIN_BRANCH = 'medic:medic:master';
+const MAIN_BRANCH = 'medic:medic:7826-dupe-public-ecr';
```

Comment:
Because the main branch isn't in the public ECR, and I removed the ECR login from the upgrade test, I switched to a clone of this branch to test the upgrade. This is temporary and just to prove that it works.